src/main/resources/data/createDbProd.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+export PGUSER=$3
+export PGPASSWORD=$4
+export PGHOST=$1
+export PGPORT=$2
+
+#create db if not exists
+psql -tc "SELECT 1 FROM pg_database WHERE datname = '$5'" | grep -q 1 | psql -c "CREATE DATABASE $5"
```

Comment:
```suggestion
psql -tc "SELECT 1 FROM pg_database WHERE datname = '$5'" | grep -q 1 | psql -c "CREATE DATABASE '$5'"
```

Also, is there a risk of SQL injection?

Reply:
This bash is just example , provided for developer experience I renamed it to be txt file , to prevent potential problems with scan